inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -166,7 +166,31 @@ void GNAModelSerial::Import(void *basePointer,
         InferenceEngine::OutputsDataMap& outputsDataMap) {
     is.exceptions(std::istream::failbit);
 
+    for (auto inputIndex = 0; inputIndex < modelHeader.nInputs; inputIndex++) {
+        uint32_t nameSize = 0;
+        readNBits<64>(nameSize, is);
+        std::vector<char> inName;
+        for (uint32_t i = 0; i < nameSize; i++) {
+            char next_char;
+            readBits(next_char, is);
+            inName.push_back(next_char);
+        }
```

Comment:
Why not:
`std::vector<char> inName(nameSize);`
`readNBytes(inName.data(), nameSize, is);`